public/css/addEventPage.css

```diff
@@ -0,0 +1,66 @@
+.addEventPageWrapper {
+    max-width: 1000px;
+    margin: 0 auto;
+}
+
+#heading {
+    width: 200px;
+    margin: 50px auto;
+    text-align: center;
+    text-transform: uppercase;
+    font-size: 2rem;
+}
+
+.eventDataRow {
+    margin-bottom: 20px;
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    height: 30px;
+}
+
+.eventDataRow
+
+.eventLabel {
+    flex: 0 0 150px;
+    border-bottom: 2px solid #56e789;
+    margin-right: 20px;
+}
+
+.eventInput {
+    flex: 1 1 250px;
+    border-radius: 5px;
+    text-decoration: none;
+    border: 1px solid;
+    padding: 5px;
+}
+
+#submitEventButton,
```

Comment:
Style on classes not on IDs.  

Also, consider renaming your id and class so that they are a single name so you do not have to use a comma